startup/41-ESM_motion.py

```diff
@@ -636,14 +636,14 @@ def move_to(self, location):
 
 #The low temperature manipulator
 
-LT_manip=ESM_motion_device(os.environ['HOME']+
-    '/.ipython/profile_collection/startup/motion_definition_files/LT_manip_definition.csv',
-    'LOW TEMPERATURE MANIPULATOR')    
+LT_manip=ESM_motion_device(os.path.join(PROFILE_STARTUP_PATH,
```

Comment:
I would tend to encourage the use of `pathlib`, but I suppose this isn't an issue if we're not demanding interoperability between OS